test-runner.js

```diff
@@ -1,5 +1,9 @@
+require('babel-polyfill');
+
 const testsContext = require.context('./tests', true, /\.js$/);
 const componentsContext = require.context('./src/', true, /components\/.*\.js$/);
+const reducersContext = require.context('./src/', true, /reducers\/.*\.js$/);
```

Comment:
I made this check for `(actions|components|containers|reducers)`. If we add another folder type we'll need to update it.